spec/docker/exec_spec.rb

```diff
@@ -138,6 +138,33 @@
       end
     end
 
+    context 'when :wait set long time value' do
+      subject {
+        described_class.create('Container' => container.id, 'Cmd' => %w[date])
+      }
+      after { container.kill!.remove }
+
+      it 'returns empty stdout and stderr messages with exitcode', :vcr do
+        expect(subject.start!(:wait => 100)).to eq([[], [], 0])
+      end
+    end
+
+    context 'when :wait set short time value' do
+      subject {
+        described_class.create(
+            'Container'    => container.id,
+            'AttachStdout' => true,
+            'Cmd'          => ['bash', '-c', 'sleep 2; echo hello']
+        )
+      }
+      after { container.kill!.remove }
+
+      it 'raises an error', :vcr do
+        pending "VCR doesn't like to record errors"
```

Comment:
VCR can't record TimeoutError so I add pending here like that.
https://github.com/swipely/docker-api/blob/bd1fc83d60d37f9fc1d9fa7e0ead4216f78e9b8d/spec/docker/container_spec.rb#L530-L534